zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/FileSystemNotebookRepo.java

```diff
@@ -1,12 +1,10 @@
 package org.apache.zeppelin.notebook.repo;
 
-import org.apache.commons.lang.StringUtils;
```

Comment:
I created https://issues.apache.org/jira/browse/ZEPPELIN-3137 for code sytle, it is better to do it in some dedicated PRs.

Reply:
Thanks -- I did a bad job cleaning up my previous commit that turned `NotebookRepo` into an abstract class. I amended the commit so that I didn't change imports in classes I didn't actually modify.